src/main/kotlin/org/rust/lang/core/completion/LookupElements.kt

```diff
@@ -308,6 +314,19 @@ private fun appendSemicolon(context: InsertionContext, curUseItem: RsUseItem?) {
     }
 }
 
+private fun addAngleBrackets(element: RsGenericDeclaration, document: Document, context: InsertionContext) {
+    if (element.typeParameters.isEmpty()) return
+
+    // do not complete angle brackets if not in type place
+    if (context.getElementOfType<RsPathExpr>() != null) return
```

Comment:
```suggestion
    val path = context.getElementOfType<RsPath>()
    if (path == null || path.parent is RsTypeReference) return
```
I'd do a reverse thing: check that we are defenitely inside a type